### DISCUSSION TITLE
Paused track-recording are not fully saved

### DISCUSSION THREAD

**Discussion by acorn371** [osmandapp/OsmAnd#12604] [Q&A]
Hi,
I have been using OsmAnd 4.0.5 on Galaxy S10e (Android 11) just for a month and this summer I had like to record my hiking tracks but I found an issue (maybe) or bad settings (by me).
Here the steps describing the issue:

1. I walk for few hours with OsmAnd (GPS active, of course) and REC on. OsmAnd is recording ....
2. I stop myself for lunch and I press "Pause" button to suspend track-recording
3. I rest for about an hour or two
4. When I decide to resume my walk I press "Resume" button to continue my way.
5. At the end of my walk I press "Finish" button and I would like to see the whole track but only the second half is shown/saved, from lunch on.

My Plugin / Trip-recording settings:
- Show start dialog (Enabled)
- Auto-recording track during navigation (Disabled)
- Logging interval during navigation (10 seconds)
- General logging interval (15 seconds)
- Minimum displacement (5m)
- Minimum accuracy (5m (16ft))
- Minimum speed (not selected)
- Auto-split recording after gap (Disabled)
- Prevent standalone logging (Disabled) <-- what does it enable/disable ? I don't understand
- Include heading (Disabled)

Thanks for any help.

Marco

**Comment by stafmans** [comment]
Hi, found your topic. I have no helping answer, but I experience something similar with 4.0.5. This week I lost 2x the last part of a track upon saving, where I am very sure the track was recorded till the very end on the moment I saved it.

**Reply by acorn371** [reply]
Hi,
I tried several times, changing my profile settings but with no solution! :-(

**Comment by vshcherb** [comment]
Do you see a separate file saved as another half track?

**Reply by acorn371** [reply]
No, I haven't a separate file. I can see the last half only.
